admin/class-convertkit-mm-admin.php

```diff
@@ -137,7 +137,7 @@ private function maybe_get_and_store_access_token() {
 			array(
 				'access_token'  => $result['access_token'],
 				'refresh_token' => $result['refresh_token'],
-				'token_expires' => ( $result['created_at'] + $result['expires_in'] ),
+				'token_expires' => ( time() + $result['expires_in'] ),
```

Comment:
Why are you switching to this? Wouldn't this set the expiration to longer than it actually is?